suppress.xml

```diff
@@ -1,48 +1,48 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-    <suppress until="2024-09-01Z">
+    <suppress until="2025-06-01Z">
         <notes><![CDATA[
    file name: jackson-databind-2.15.3.jar
```

Comment:
/do we still not have this available?
if a latest release is available for these lets try them out.
but leave kafka clients as they may not work.

Reply:
Great suggestion. I have looked into all of the suppressions individually and you were right. When we updated the all of the versions etc about a month ago, we actually took care of most of these. So I was able to delete most of the suppressions now.

I left the suppressions for kafka-clients.

I also left two suppressions that are sub-dependencies of email-producer. We have the most up-to-date email-producer though.